src/uwtools/drivers/driver.py

```diff
@@ -154,6 +123,133 @@ def _driver_name(self) -> str:
         Returns the name of this driver.
         """
 
+    @property
+    def _resources(self) -> Dict[str, Any]:
+        """
+        Returns configuration data for the runscript.
```

Comment:
Should we have anything related to a runscript in the base class? It seems to me that that would belong in the subclass that deals with standalone execution. Would platform information be a concern for an assets-only driver? Maybe you're aware of good reasons I'm not thinking of...

I do get that `run_dir()` needs to be in the base class, so that we have a directory to place assets in.

Reply:
Hm, maybe this is actually a bad doc string? The resources function just keeps the account and rundir info from the platform block, I think this is probably ok for the base if I fix the name

Reply:
I've updated the text, but I can shift it still if that's whats wanted

Reply:
Am I right in thinking that we still need to deal with `platform` here because it's always checked by the `_validate()` method when a driver object is instantiated? I'm inclined to leave things as they are now, but feel a little bit uneasy since I haven't convinced myself that an assets-only driver needs to know platform details. If I could think of an example, I'd be satisfied that it should remain here. I think an alternative might be to remove `platform` from this `Assets` method (it's already included in `Driver`'s `_resources()` method), remove it from the `Assets` `_validate()` method, and have `Driver` implement its own `_validate()`, which would be identical to the one currently in `Assets` -- i.e. it would validate both the driver block and the `platform` block. If anyone can think of a single example where an `Assets` driver needs platform information, I'd drop it; but as thing stand, it feels weird to me to see platform information here. Anyone else have thoughts on this?

Reply:
Mm, that isn't unfair - thinking of this more as one block of a series of some future `experiment` key, the experiment is probably where this should reside

Reply:
Exactly. If this assets driver is called upon by a coupled-component driver, I think that latter driver (which would subclass `Driver`) would be where platform-specific considerations would arise.

Unless our currently one-and-only assets driver (WW3) has some use for platform info -- which would be the proof of existence I'm looking for -- I'd err on the side of making `Asset` ignorant of `platform`. We can reintroduce it later if we need to.